.vscode/settings.json

```diff
@@ -2,5 +2,5 @@
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "[typescript]": {
     "typescript.preferences.importModuleSpecifier": "relative"
-  },
```

Comment:
Are the changes here required? Same for `.yarnrc.yml` and `tsconfig.json`.

The prettier rule should only apply to Typescript files 🤔

Reply:
Not strictly required, but I ran prettier in the project root so several non-TS files were formatted. I think these changes are good (JSONs trailing commas are disallowed, for example), so I included them. No strong opinion on it tough, I can keep just the TS changes if you prefer 🙂

Reply:
> They aren't strictly required. I ran prettier in the project root so several non-TS files were formatted. I think these changes are good (JSONs trailing commas are disallowed, for example)

Then would you enable the validation on JSON and YML files? (to avoid the example that you mentioned)

Reply:
No strong opinion on that, but I think it might be nice. We could update the `package.json` tasks accordingly in a separate PR if you agree.

Reply:
I totally agree. I was reading the prettier docs, and it seems that by default it inherits from version control files (i.e. `.gitignore`) to compose the ignore list.

> By default prettier ignores files in version control systems directories (".git", ".svn" and ".hg") and node_modules (if [--with-node-modules CLI option](https://prettier.io/docs/en/cli#--with-node-modules) not specified)

So, by taking a look at the list of prettier inspected files, I'd say the only files not included in `.gitignore` we would like to avoid _prettifying_ are the ones contained in `/.yarn` (markdown is also supported by prettier). So if you agree I'd be happy to open a PR including a `.prettierignore`file and removing the regexes from `package.json` 🙂